engine/table/src/main/java/io/deephaven/stream/TablePublisher.java

```diff
@@ -169,6 +169,27 @@ public boolean isAlive() {
         return adapter.isAlive();
     }
 
+    /**
+     * The {@link Table#BLINK_TABLE_ATTRIBUTE blink table} with its {@link Table#getAttribute(String) attribute}
+     * {@value Table#INPUT_TABLE_ATTRIBUTE} set to an {@link io.deephaven.engine.util.input.InputTableUpdater}
+     * implementation based on {@code this}. This is primarily useful for existing code that already works with
+     * {@link io.deephaven.engine.util.input.InputTableUpdater} - new code should probably prefer to work directly with
+     * {@code this}.
+     *
+     * <p>
+     * May return {@code null} if invoked more than once and the initial caller does not enforce strong reachability of
```

Comment:
The `InputTableUpdater` you're providing doesn't match the usual semantics (blocking until published). You may want to document this.